### ISSUE TITLE
macos/vbox: waiting for kube-proxy to be up for configmap update: timed out

### ISSUE THREAD

**Issue by rbizzell40** [kubernetes/minikube#3605]
**Is this a BUG REPORT or FEATURE REQUEST?** (Bug Report):

Please provide the following details:

**Environment**:
MacOS Mojave Version 10.14.2
**Minikube version** (use `minikube v0.31.0`):
- **OS** (e.g. from /etc/os-release):
- **VM Driver** (e.g. `cat ~/.minikube/machines/minikube/config.json | grep DriverName`): virtualbox
- **ISO version** (e.g. `cat ~/.minikube/machines/minikube/config.json | grep -i ISO` or `minikube ssh 
 cat /etc/VERSION`): minikube-v0.33.1.iso  v0.33.1
*Install tools**:
- **Others**:
The above can be generated in one go with the following commands (can be copied and pasted directly into your terminal):
```
minikube version
echo "";
echo "OS:";
cat /etc/os-release
echo "";
echo "VM driver:"; 
grep DriverName ~/.minikube/machines/minikube/config.json
echo "";
echo "ISO version";
grep -i ISO ~/.minikube/machines/minikube/config.json
```

**What happened**:
E0129 13:48:41.133127   90823 start.go:210] Error parsing version semver:  Version string empty
E0129 14:10:43.188119   92531 start.go:348] Error restarting cluster:  restarting kube-proxy: waiting for kube-proxy to be up for configmap update: timed out waiting for the condition
What you expected to happen**:
I expected the cluster to start up as normal

**How to reproduce it** (as minimally and precisely as possible):

**Output of `minikube logs` (if applicable)**:

**Anything else do we need to know**:
No

**Comment by tstromberg** [comment]
Sorry to hear that minikube didn't work on your host. Does this machine happen to have any kind of VPN or firewall software installed? 

If you don't mind, the following info would be very helpful in sorting out what's going on:

- Command-line options you may have provided to `minikube start`, if any.
- Cut & paste of the full text output of `minikube start`
- Attach the output of: 
`minikube ssh 'docker logs $(docker ps -a -f name=k8s_kube-api --format={{.ID}})' | tee ~/Desktop/apiserver_log.txt`

Thank you for the feedback! We really appreciate it.

**Comment by tstromberg** [comment]
This is possibly a dupe of #3354

**Comment by rbizzell40** [comment]
Ok I will get this info soon as I get home

**Comment by tstromberg** [comment]
Thanks!

**Comment by rbizzell40** [comment]
E0129 18:52:58.676970    8342 start.go:302] Error restarting cluster:
restarting
kube-proxy: waiting for kube-proxy to be up for configmap update: timed out
waiting for the condition

================================================================================

An error has occurred. Would you like to opt in to sending anonymized crash

information to minikube to help prevent future errors?

To opt out of these messages, run the command:

minikube config set WantReportErrorPrompt false

================================================================================

Please enter your response [Y/n]: %






minikube ssh 'docker logs $(docker ps -a -f name=k8s_kube-api
--format={{.ID}})' | tee ~/Desktop/apiserver_log.tx

Flag --admission-control has been deprecated, Use
--enable-admission-plugins or --disable-admission-plugins instead. Will be
removed in a future version.

Flag --insecure-port has been deprecated, This flag will be removed in a
future version.

I0129 23:54:30.880100       1 server.go:135] Version: v1.10.0

I0129 23:54:30.880376       1 server.go:679] external host was not
specified, using 192.168.99.100

failed to create listener: failed to listen on 0.0.0.0:8443: listen tcp
0.0.0.0:8443: bind: address already in use

➜  .kube minikube status

E0129 18:55:14.774798   10456 status.go:85] Error cluster status: getting
status: running command: sudo systemctl is-active kubelet

.: Process exited with status 3

**Comment by tstromberg** [comment]
Sorry about this not working well. 

I believe this was fixed in minikube v0.35.0. In some cases, a different error may appear, but it should no longer be blocking on configmap updates.
